src/components/NurseryWithdrawals/NurseryWithdrawalsTable.tsx

```diff
@@ -136,21 +141,33 @@ export default function NurseryWithdrawalsTable(): JSX.Element {
       };
       searchValueChildren.push(fromNurseryNode);
 
-      const destinationNurseryNode: FieldNodePayload = {
+      const speciesNameNode: FieldNodePayload = {
         operation: 'field',
-        field: 'destinationName',
+        field: 'batchWithdrawals.batch_species_scientificName',
         type: 'Fuzzy',
         values: [debouncedSearchTerm],
       };
-      searchValueChildren.push(destinationNurseryNode);
+      searchValueChildren.push(speciesNameNode);
 
-      const speciesNameNode: FieldNodePayload = {
+      if (!selectedPlantingSite) {
```

Comment:
do we need to check || selectedPlantingSite.id === -1?

Reply:
No, this is to add the destinationName to the query when we don't pass a selectedPlantingSite, that is for v1 only. If selected site is -1, we don't want to filter by destinationName at all

Reply:
got it 👍